cmd/kubebuilder/docs/docs.go

```diff
@@ -157,13 +158,21 @@ func RunDocs(cmd *cobra.Command, args []string) {
 	generatecmd.RunGenerate(cmd, args)
 
 	if brodocs {
+		user, err := user.Current()
+		if err != nil {
+			log.Fatalf("error: %v\n", err)
+		}
+
 		// Run the docker command to build the docs
 		c = exec.Command("docker", "run",
+			"-u", fmt.Sprintf("%s:%s", user.Uid, user.Gid),
+			"-v", "/etc/group:/etc/group:ro",
+			"-v", "/etc/passwd:/etc/passwd:ro",
 			"-v", fmt.Sprintf("%s:%s", filepath.Join(wd, outputDir, "includes"), "/source"),
 			"-v", fmt.Sprintf("%s:%s", filepath.Join(wd, outputDir, "build"), "/build"),
 			"-v", fmt.Sprintf("%s:%s", filepath.Join(wd, outputDir, "build"), "/build"),
 			"-v", fmt.Sprintf("%s:%s", filepath.Join(wd, outputDir), "/manifest"),
-			"gcr.io/kubebuilder/brodocs",
+			"gcr.io/kubebuilder/brodocs:v0.1.11",
```

Comment:
should this version tag be hard coded?

Reply:
Yes I think so.  It makes it so we don't need to test each brodocs version with each kubebuilder release.  If we just use latest, we would have to test each brodocs image with each kubebuilder release, or risk breaking old kubebuilder versions.

Reply:
I was thinking more along the lines of using something like kubeBuilderVersion from cmd/kubebuilder/version, falling back to latest if that's not defined

Reply:
I guess something that could be done after this patch